Yafc.UI/Core/Ui.cs

```diff
@@ -167,21 +181,25 @@ public static void ProcessEvents() {
                         }
 
                         break;
+
                     case SDL.SDL_EventType.SDL_RENDER_TARGETS_RESET:
                         break;
+
                     case SDL.SDL_EventType.SDL_USEREVENT:
                         if (evt.user.type == asyncCallbacksAdded) {
                             ProcessAsyncCallbackQueue();
                         }
 
                         break;
```

Comment:
Again a lot of double whitelines. One should be enough to break the block. Either keep the break with the code above or below. I would prefer to not use whilteline at all around the break, the break itself already stands out quite well.

Reply:
The blank line above the `break` is due to `if` ending. I'll take a look at breaks again tomorrow with a fresh look, to see if I overdid blank lines.